### PR TITLE
feat: add far group example for toolbar

### PR DESCRIPTION
**Pull request by chpalac**
## Overview

Fixes: #25804

Adding the far group example for toolbar

Screenshot (5)

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `59.018 kB`<br />`16.371 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `186.456 kB`<br />`52.327 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |   `33.48 kB`<br />`11.037 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `5.857 kB`<br />`1.978 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/1bff9a3148961667fe516b46fead5798db794ae6'>1bff9a3148961667fe516b46fead5798db794ae6</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/Persona-mount_base.html">665</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/Persona-mount.html">3676</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/Persona-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/Avatar-mount_base.html">1578</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/Avatar-mount.html">1550</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/Button-mount_base.html">1137</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/Button-mount.html">1158</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/FluentProvider-mount_base.html">1936</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/FluentProvider-mount.html">1916</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/FluentProviderWithTheme-mount_base.html">765</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/FluentProviderWithTheme-mount.html">791</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">720</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">709</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">743</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">771</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/MakeStyles-mount_base.html">2314</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/MakeStyles-mount.html">2320</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/Persona-mount_base.html">665</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/Persona-mount.html">3676</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/Persona-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/SpinButton-mount_base.html">3067</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25874/perf-test-react-components/SpinButton-mount.html">3086</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by chpalac** [comment]
@ling1726 When you can, please have a look into this one? 🙏🏻

File: change/@fluentui-react-toolbar-1e4b5339-c938-49e8-992a-00afb2aa131f.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "chore: create group",
```
Comment (ling1726) [inline change/@fluentui-react-toolbar-1e4b5339-c938-49e8-992a-00afb2aa131f.json:3]:
```suggestion
  "comment": "feat: Implement `ToolbarGroup` component",
```

File: packages/react-components/react-toolbar/src/stories/Toolbar/ToolbarFarGroup.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <ToolbarGroup role="presentation">
+        <ToolbarButton aria-label="Increase Font Size" appearance="primary" icon={<FontIncrease24Regular />} />
+        <ToolbarButton aria-label="Decrease Font Size" icon={<FontDecrease24Regular />} />
+        <ToolbarButton aria-label="Reset Font Size" icon={<TextFont24Regular />} />
+        <ToolbarDivider />
+        <ToolbarButton aria-label="Increase Font Size" appearance="primary" icon={<FontIncrease24Regular />} />
+        <ToolbarButton aria-label="Decrease Font Size" icon={<FontDecrease24Regular />} />
+        <ToolbarButton aria-label="Reset Font Size" icon={<TextFont24Regular />} />
+      </ToolbarGroup>
+      <ToolbarGroup role="presentation">
```
Comment (ling1726) [inline packages/react-components/react-toolbar/src/stories/Toolbar/ToolbarFarGroup.stories.tsx:26]: let's add `role="presentation"` directly in the component

File: packages/react-components/react-toolbar/src/stories/Toolbar/ToolbarFarGroup.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <ToolbarButton aria-label="Increase Font Size" appearance="primary" icon={<FontIncrease24Regular />} />
+        <ToolbarButton aria-label="Decrease Font Size" icon={<FontDecrease24Regular />} />
+        <ToolbarButton aria-label="Reset Font Size" icon={<TextFont24Regular />} />
```
Comment (ling1726) [inline packages/react-components/react-toolbar/src/stories/Toolbar/ToolbarFarGroup.stories.tsx:29]: don't use sized icons in examples, we want to recommend users to use the unsized icons

File: packages/react-components/react-toolbar/src/stories/Toolbar/ToolbarFarGroup.stories.tsx (hunk 2 above)
Comment (chpalac) [reply packages/react-components/react-toolbar/src/stories/Toolbar/ToolbarFarGroup.stories.tsx:26]: Added in the `useToolbarGroup` https://github.com/microsoft/fluentui/pull/25874/commits/57fc7f011a622a8041d0dd832350b80566dac922

**Review by ling1726 — APPROVED** [review]

File: apps/vr-tests-react-components/package.json [hunk 4]
```diff
@@ -29,6 +29,7 @@
     "@fluentui/react-card": "9.0.0-beta.33",
     "@fluentui/react-checkbox": "^9.0.13",
     "@fluentui/react-combobox": "^9.0.0-beta.16",
+    "@fluentui/react-components": "^9.7.1",
```
Comment (ling1726) [inline apps/vr-tests-react-components/package.json:32]: remove this please

**Pull request merged**